src/db/connection.py

```diff
@@ -0,0 +1,3 @@
+def getConnection():
```

Comment:
Python style is to use camelCase, so can you make this get_connection?

Reply:
That's not true.

```py
def getConnection(): # This is camelCase

def get_connection(): # This is called snake_case
```

Reply:
Ah I messed that up. I did mean snake_case